connect/resources/product.py

```diff
@@ -0,0 +1,77 @@
+import json
+from .base import BaseResource
+
+
+class ProductsResource(BaseResource):
+    """ Allows listing and obtaining several types of objects.
+        :param Config config: Config object or ``None`` to use environment config (default).
+    """
+    resource = 'products'
+
+    def list_parameters(self, product_id):
+        """ List parameters for a product.
+        :param str product_id: Primary key of the product to search for.
+        :return: response object with templates contents.
+        """
+        """
+        # type: (Dict[str, Any]) -> List[Any]
+        """
+        response, _ = self._api.get(
+            '/public/v1/products/' + product_id + '/parameters/'
+        )
+        response = json.loads(response)
+        return response
+
+    def create_parameter(self, product_id, body):
+        """ Create parameter for a product.
+        :param str product_id: Primary key of the product to create parameter.
+        :param str body: Body of the parameter to create.
+        :return: response object with templates contents.
+        """
+        """
+        # type: (Dict[str, Any]) -> List[Any]
+        """
+        if not product_id:
+            raise ValueError('Invalid ID')
+        path = '/public/v1/products/' + product_id + '/parameters/'
+        response = self._api.post(
+            path=path,
+            json=body
+        )
+        return response
+
+    def update_parameter(self, product_id, parameter_id, body):
+        """ Update parameter for a product.
+        :param str product_id: Primary key of the product to update parameter.
+        :param str parameter_id: Primary key of the parameter to update.
+        :param str body: Body of the parameter to update.
+        :return: response object with templates contents.
+        """
+        """
+        # type: (Dict[str, Any]) -> List[Any]
+        """
+        if not product_id:
+            raise ValueError('Invalid ID')
+        path = '/public/v1/products/' + product_id + '/parameters/' + parameter_id
```

Comment:
I think it is better to use [urljoin](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin)